src/pages/EarnStrategies/components/form/EarnStrategyDepositForm.tsx

```diff
@@ -0,0 +1,66 @@
+import { newMonetaryAmount } from '@interlay/interbtc-api';
```

Comment:
I would just follow the same file structure as in our other pages folders, meaning that you should bring the component into his own folder

Reply:
Also, you are missing the index file under the components folder. This `index` folder should only export whatever is going to be needed on the EarnStrategy page file